### PR TITLE
multi_disk:fix regain disk driver after booting

### PR DESCRIPTION
**Pull request by qingwangrh**
The disks driver are not always same after booting. 
It needs to regain the disks driver after booting.
ID:2158316
Signed-off-by: qingwangrh <qinwang@redhat.com>

### CODE REVIEW

**Comment by qingwangrh** [comment]
python ConfigTest.py --testcase=multi_disk --iothread_scheme=roundrobin --nr_iothreads=2 --platform=x86_64 --guestname=RHEL.9.2.0,Win2019 --driveformat=virtio_scsi --nicmodel=virtio_net --imageformat=qcow2 --machines=q35 --customsparams="vm_mem_limit = 12G\nimage_aio=threads" --firmware=default_bios --clone=no --netdst=virbr0

 (01/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.one_disk_repeat.q35: STARTED
 (01/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.one_disk_repeat.q35: PASS (99.22 s)
 (02/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.max_disk.q35: STARTED
 (02/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.max_disk.q35: PASS (410.58 s)
 (03/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.all_drive_format_types.q35: STARTED
 (03/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.all_drive_format_types.q35: PASS (67.49 s)
 (04/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.block.q35: STARTED
 (04/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.block.q35: PASS (113.82 s)
 (05/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.generic.with_cache_writethrough.q35: STARTED
 (05/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.generic.with_cache_writethrough.q35: PASS (112.10 s)
 (06/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.generic.with_cache_writeback.q35: STARTED
 (06/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.generic.with_cache_writeback.q35: PASS (112.45 s)
 (07/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_lun.q35: STARTED
 (07/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_lun.q35: PASS (851.13 s)
 (08/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_scsiid_lun.q35: STARTED
 (08/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_scsiid_lun.q35:PASS (336.80 s)
 (09/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_bus_scsiid_lun.q35: STARTED
 (09/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_bus_scsiid_lun.q35: PASS (442.88 s)
 (10/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk.one_disk_repeat.q35: STARTED
 (10/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk.one_disk_repeat.q35: PASS (208.51 s)
 (11/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk.max_disk.q35: STARTED
 (11/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk.max_disk.q35: PASS (928.92 s)
 (12/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk.all_drive_format_types.q35: STARTED
 (12/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk.all_drive_format_types.q35: PASS (137.46 s)
 (13/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.block.q35: STARTED
 (13/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.block.q35: PASS (229.76 s)
 (14/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.generic.with_cache_writethrough.q35: STARTED
 (14/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.generic.with_cache_writethrough.q35: PASS (225.83 s)
 (15/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.generic.with_cache_writeback.q35: STARTED
 (15/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.generic.with_cache_writeback.q35: PASS (231.34 s)
 (16/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_lun.q35: STARTED
 (16/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_lun.q35: PASS (755.81 s)
 (17/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_scsiid_lun.q35: STARTED
 (17/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_scsiid_lun.q35: PASS (694.74 s)
 (18/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_bus_scsiid_lun.q35: STARTED
 (18/18) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_bus_scsiid_lun.q35: PASS (797.13 s)
RESULTS    : PASS 18 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0

**Comment by qingwangrh** [comment]
depends on https://github.com/avocado-framework/avocado-vt/pull/3620

**Comment by qingwangrh** [comment]
@YongxueHong @XueqiangWei @zhenyzha Please help to review, thanks.

**Comment by zhenyzha** [comment]
Hit a problem unrelated to this patch
When only nvme host runs the cases, the error is as follows. Do we have plans to optimize this problem? 
another code ok for me.


```
 (1/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.one_disk_repeat.arm64-pci: PASS (207.90 s)
 (2/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.max_disk.arm64-pci: STARTED
 (2/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.max_disk.arm64-pci: PASS (1307.51 s)
 (3/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.all_drive_format_types.arm64-pci: STARTED
 (3/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.all_drive_format_types.arm64-pci: SKIP: usb controller usb-ehci not available
 (4/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.block.arm64-pci: STARTED
 (4/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.block.arm64-pci: ERROR: Matching disk is in blacklist. name = '/dev/sd*', matching = '['', '/dev/sda', '/dev/sdb', '/dev/sdc', '/dev/sdd', '/dev/sde', '/dev/sdf', '/dev/sdg', '/dev/sdh', '/dev/sdi']' and selector = '-9' (8.38 s)
 (5/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.generic.with_cache_writethrough.arm64-pci: STARTED
 (5/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.generic.with_cache_writethrough.arm64-pci: ERROR: Matching disk is in blacklist. name = '/dev/sg*', matching = '['', '/dev/sg0', '/dev/sg1', '/dev/sg2', '/dev/sg3', '/dev/sg4', '/dev/sg5', '/dev/sg6', '/dev/sg7', '/dev/sg8']' and selector = '-9' (8.61 s)
 (6/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.generic.with_cache_writeback.arm64-pci: STARTED
 (6/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.generic.with_cache_writeback.arm64-pci: ERROR: Matching disk is in blacklist. name = '/dev/sg*', matching = '['', '/dev/sg0', '/dev/sg1', '/dev/sg2', '/dev/sg3', '/dev/sg4', '/dev/sg5', '/dev/sg6', '/dev/sg7', '/dev/sg8']' and selector = '-9' (8.66 s)
 (7/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_lun.arm64-pci: STARTED
 (7/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_lun.arm64-pci: PASS (1201.58 s)
 (8/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_scsiid_lun.arm64-pci: STARTED
 (8/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_scsiid_lun.arm64-pci: PASS (513.17 s)
 (9/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_bus_scsiid_lun.arm64-pci: STARTED
 (9/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_bus_scsiid_lun.arm64-pci: PASS (653.06 s)
RESULTS    : PASS 5 | ERROR 3 | FAIL 0 | SKIP 1 | WARN 0 | INTERRUPT 0 | CANCEL 0
 
```

**Comment by qingwangrh** [comment]
> Hit a problem unrelated to this patch When only nvme host runs the cases, the error is as follows. Do we have plans to optimize this problem? another code ok for me.
> 
> ```
>  (1/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.one_disk_repeat.arm64-pci: PASS (207.90 s)
>  (2/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.max_disk.arm64-pci: STARTED
>  (2/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.max_disk.arm64-pci: PASS (1307.51 s)
>  (3/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.all_drive_format_types.arm64-pci: STARTED
>  (3/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.all_drive_format_types.arm64-pci: SKIP: usb controller usb-ehci not available
>  (4/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.block.arm64-pci: STARTED
>  (4/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.block.arm64-pci: ERROR: Matching disk is in blacklist. name = '/dev/sd*', matching = '['', '/dev/sda', '/dev/sdb', '/dev/sdc', '/dev/sdd', '/dev/sde', '/dev/sdf', '/dev/sdg', '/dev/sdh', '/dev/sdi']' and selector = '-9' (8.38 s)
>  (5/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.generic.with_cache_writethrough.arm64-pci: STARTED
>  (5/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.generic.with_cache_writethrough.arm64-pci: ERROR: Matching disk is in blacklist. name = '/dev/sg*', matching = '['', '/dev/sg0', '/dev/sg1', '/dev/sg2', '/dev/sg3', '/dev/sg4', '/dev/sg5', '/dev/sg6', '/dev/sg7', '/dev/sg8']' and selector = '-9' (8.61 s)
>  (6/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.generic.with_cache_writeback.arm64-pci: STARTED
>  (6/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.generic.with_cache_writeback.arm64-pci: ERROR: Matching disk is in blacklist. name = '/dev/sg*', matching = '['', '/dev/sg0', '/dev/sg1', '/dev/sg2', '/dev/sg3', '/dev/sg4', '/dev/sg5', '/dev/sg6', '/dev/sg7', '/dev/sg8']' and selector = '-9' (8.66 s)
>  (7/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_lun.arm64-pci: STARTED
>  (7/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_lun.arm64-pci: PASS (1201.58 s)
>  (8/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_scsiid_lun.arm64-pci: STARTED
>  (8/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_scsiid_lun.arm64-pci: PASS (513.17 s)
>  (9/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_bus_scsiid_lun.arm64-pci: STARTED
>  (9/9) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_bus_scsiid_lun.arm64-pci: PASS (653.06 s)
> RESULTS    : PASS 5 | ERROR 3 | FAIL 0 | SKIP 1 | WARN 0 | INTERRUPT 0 | CANCEL 0
>  
> ```

I suggest opening an auto bug to track this.

**Review by zhenyzha — APPROVED** [review]
LGTM.Acked-by: Zhenyu Zhang <zhenyzha@redhat.com>

**Comment by XueqiangWei** [comment]
Tested with windows and linux guests, the results were passed.
(1/9) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.one_disk_repeat.q35: PASS (147.68 s)
(2/9) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.max_disk.q35: PASS (918.31 s)
 3/9) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.all_drive_format_types.q35: PASS (91.78 s)
(4/9) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.block.q35: PASS (159.29 s)
 (5/9) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.generic.with_cache_writethrough.q35: PASS (158.07 s)
 (6/9) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.generic.with_cache_writeback.q35: PASS (157.49 s)
 (7/9) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_lun.q35: PASS (871.54 s)
(8/9) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_scsiid_lun.q35: PASS (363.89 s)
(9/9) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_bus_scsiid_lun.q35: PASS (472.07 s)
(1/9) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.multi_disk.one_disk_repeat.q35: PASS (207.90 s)
(2/9) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.multi_disk.max_disk.q35: PASS (1365.28 s)
(3/9) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.multi_disk.all_drive_format_types.q35: PASS (160.36 s)
(4/9) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.block.q35: PASS (281.94 s)
(5/9) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.generic.with_cache_writethrough.q35: PASS (282.95 s)
(6/9) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.generic.with_cache_writeback.q35: PASS (280.20 s)
(7/9) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_lun.q35: PASS (783.36 s)
(8/9) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_scsiid_lun.q35: PASS (730.77 s)
(9/9) Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.multi_disk.virtio_scsi_variants.multi_bus_scsiid_lun.q35: PASS (796.58 s)

**Comment by XueqiangWei** [comment]
LGTM,
Acked-by: xuwei@redhat.com

**Comment by qingwangrh** [comment]
Updated as discussion result.

File: qemu/tests/multi_disk.py [hunk 1]
```diff
@@ -246,28 +303,7 @@ def _do_post_cmd(session):
...
-                test.fail("Fail to list all the volumes"
-                          ", %s" % err_msg % len(disks))
-            if len(disks) > drive_letters:
-                black_list.extend(utils_misc.get_winutils_vol(session))
-                disks = random.sample(disks, drive_letters - len(black_list))
-            error_context.context("Clear readonly for all disks and online "
-                                  "them in windows guest.", test.log.info)
-            if not utils_disk.update_windows_disk_attributes(session, disks):
-                test.fail("Failed to update windows disk attributes.")
-            dd_test = "no"
```
Comment (XueqiangWei) [inline qemu/tests/multi_disk.py:263]: Do you missed this line? And Could you please add the latest test results. Thanks.

Comment (qingwangrh) [reply qemu/tests/multi_disk.py:263]: updated

**Comment by qingwangrh** [comment]
Pass test  with follow test
python ConfigTest.py --testcase=multi_disk --iothread_scheme=roundrobin --nr_iothreads=2 --platform=x86_64 --guestname=RHEL.9.2.0,Win2019 --driveformat=virtio_scsi --nicmodel=virtio_net --imageformat=qcow2 --machines=q35 --customsparams="vm_mem_limit = 12G\nimage_aio=threads" --firmware=default_bios --clone=no --netdst=virbr0

**Review by YongxueHong — APPROVED** [review]
LGTM.

**Pull request merged**